docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -117,6 +117,11 @@ If you have a use case that you'd like to share, feel free to contribute to this
 1. Add dashboards to each folder.
 1. Use folder permissions to add US-based users as Editors to the `US` folder and assign EU-based users as Editors to the `EU` folder.
 
+### Assign a user specific set of roles
+
+1. Create a user with the `No Basic Role` role.
```

Comment:
Nit: not sure how to best phrase this, but  "`No Basic Role` role" sounds weird to me.
```suggestion
1. Create a user with the `No Basic Role` selected under organization roles.
```